BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -129,6 +126,17 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate,
         cell.storeImage.kf.setImage(with: url, placeholder: nil, options: options, progressBlock: nil, completionHandler: nil)
         
     }
+    
+    func goCombineAndLoadAddress(into cell: StoresTableViewCell, withStore address: NSDictionary) {
+        
+        guard let street = address["address_line_1"] as? String else { return }
```

Comment:
To be honest, the Address should have its own Data Structure, and a Store should have an Address type.
That is better than constantly having to pull out the values from the dictionary each time.

Hit me up if you need clarification on this.